Fix builds for Ubuntu 22.10

We trigger PPA release builds for the upcoming Ubuntu release as early failure indicator and in the last release it actually failed:
https://launchpadlibrarian.net/617180489/buildlog_ubuntu-kinetic-amd64.solc_1%3A0.8.16-0ubuntu1~kinetic_BUILDING.txt.gz

We should check with a ubuntu 22.10 docker image what to do to fix those builds. Best asap, but definitely before its release in October.

Should maybe have fixed this before this release - but yeah, this was probably the last release in which we'll get away with saying "22.10 is not released yet anyways" :-).

Not entirely sure if we should actually add CI runs for the short-lived ubuntu releases...

Anyways, new log, different failure this time:

https://launchpadlibrarian.net/622411533/buildlog_ubuntu-kinetic-amd64.solc_1:0.8.17-0ubuntu1~kinetic_BUILDING.txt.gz

Seems to be fmtlib related:

```
CMake Error at liblangutil/CMakeLists.txt:34 (target_link_libraries):
  Target "langutil" links to:

    fmt::fmt-header-only

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
```

Even though fmtlib should be bundled in the PPA source archives...

--- a/scripts/release_ppa.sh
+++ b/scripts/release_ppa.sh
@@ -217,7 +217,7 @@ override_dh_shlibdeps:
 	dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info
 
 override_dh_auto_configure:
-	dh_auto_configure -- -DTESTS=OFF ${CMAKE_OPTIONS}
+	dh_auto_configure -- -DTESTS=OFF -DFETCHCONTENT_FULLY_DISCONNECTED=OFF ${CMAKE_OPTIONS}
 EOF
 cat <<EOF > debian/copyright
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/

